qiskit/transpiler/target.py

```diff
@@ -979,7 +981,7 @@ def instruction_properties(self, index):
             InstructionProperties: The instruction properties for the specified instruction tuple
         """
         instruction_properties = [
-            inst_props for op in self._gate_map for _, inst_props in self._gate_map[op].items()
+            inst_props for _, qargs in self._gate_map.items() for _, inst_props in qargs.items()
```

Comment:
Rather than `for _, x in y.items()`, we might consider using `for x in y.values()`?